sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -322,9 +322,9 @@ object FunctionRegistry {
     expression[CurrentDate]("current_date"),
     expression[CurrentTimestamp]("current_timestamp"),
     expression[DateDiff]("datediff"),
-    expression[DateAdd]("date_add"),
+    expression[AddDays]("date_add"),
```

Comment:
I may understand why hive call it `date_add`...

Other databases also have `date_add`, but it takes interval type, which means it can handle days or months or both.

To keep hive-compatibility, we should keep it as it was, but rename the expression class name LGTM